internal/cli/cmd/connectivity.go

```diff
@@ -87,9 +90,29 @@ func newCmdConnectivityTest() *cobra.Command {
 				cancel()
 			}()
 
-			if err := connectivity.Run(ctx, cc); err != nil {
-				fatalf("Connectivity test failed: %s", err)
+			done := make(chan bool)
```

Comment:
This doesn't need to be `chan bool` and instead can be a "signal" channel, aka `chan struct{}`

Reply:
Sure, that eliminates a single alloc. :stuck_out_tongue: Will fix, thanks.